src/dev/flang/ast/Call.java

```diff
@@ -1344,7 +1344,9 @@ private void resolveFormalArg(Resolution res, int argnum, AbstractFeature frml,
                 Generic g = frmlT.genericArgument();
                 var frmlTs = g.replaceOpen(g.feature() == _calledFeature
                                            ? _generics
-                                           : _target.type().generics());
+                                           : (heir.isGenericArgument() // see for example #1919
+                                                ? heir.genericArgument().constraint(res, context).generics()
+                                                : heir.generics()));
```

Comment:
```suggestion
                                           : (heir.isGenericArgument() // see for example #1919
                                                ? heir.genericArgument().constraint(res, context)
                                                : heir
                                             ).generics());
```
The pattern `t.isGenericArgument() ? t.genericArgument.constraint(res, context) : t` occurs at several places. Might be good to move this in to a method in `AbstractType`, in a separate PR.